shared/gh/js/controllers/gh.api.util.js

```diff
@@ -161,6 +161,33 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
     };
 
     /**
+     * Get the Date of the first day of the specified term
+     *
+     * @param  {String}    termName    The name of the term to get the date of the first day for
+     * @return {Date}                  The date of the first day of the term
+     */
+    var getFirstDayOfTerm = exports.getFirstDayOfTerm = function(termName) {
+        if (!_.isString(termName)) {
+            throw new Error('A valid term name should be provided');
+        }
+
+        // Get the configuration
+        var config = require('gh.core').config;
+        // Get the correct terms associated to the current application
+        var terms = config.terms[config.academicYear];
+        // Find the term to get the first day's date for
+        var term =_.find(terms, function(term) {
+            if (term.name === termName) {
```

Comment:
It looks like the coverage is decreased since there is no `else` clause to be executed